README.md

```diff
@@ -241,3 +241,56 @@ docker run                         \
 curl http://localhost:8001/
 ```
 
+### Virtual Machine that allow create docker components for debugging purpose
+
+Step-by-step instruction to building and running virtual machine with all needed dockers.
+
+- Install require libs:
```

Comment:
I need to write more detailed instructions. Please remind me to give you text for the README before you merge.

Reply:
Done.